app/src/main/res/values/strings.xml

```diff
@@ -20,6 +20,9 @@
 
     <string name="google_search">Searching the query from google</string>
     <string name="set_alarm">Setting the Alarm</string>
+    <string name="reminderDes">Write the description for the Reminder</string>
```

Comment:
We are currently using snake_case string names can you do the same, I know it's silly but it unifies the code.